src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
@lucenticus I have a couple of thoughts:
* When I have briefly discussed this offline with @jkotas, he has said that he thinks the m_cbActualArgSize should actually be the stack arg size of the native caller after all, so no augmenting should be necessary and it is possible that there is some other issue with the [NativeCallable]
* Could you please dump the the disassembly of the IL_STUB_ReversePInvoke stub for the DoCallBack_MarshalRefInt_Stdcall test? That should confirm what arguments the stub really expects.